src/Authentication/AuthenticationStrategy.cs

```diff
@@ -0,0 +1,5 @@
+namespace Microsoft.Graph.Cli.Authentication;
+
+public enum AuthenticationStrategy {
+    DeviceCode
```

Comment:
I believe that if you add doc comments on the enum entries, they'll flow naturally to the help command

Reply:
I'll test this

Reply:
This doesn't seem to be the case.